webui/react/src/components/CheckpointModal.tsx

```diff
@@ -58,7 +59,9 @@ const getStorageLocation = (
       }
       break;
     case CheckpointStorageType.DIRECTORY:
-      // unsupported, natch
+      if (container_path) {
+        location = `file:/${container_path}`;
```

Comment:
**question:** just confirming that `file:/` is the right prefix, since in the lines above, they are using `file://` (two forward slashes).

Reply:
Good catch, the container path can be absolute or relative path